Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -351,19 +339,23 @@ def reset_last_run():
 
 def main() -> None:
     params = demisto.params()
-    client_id = params.get('client_id')
-    client_secret = params.get('client_secret')
+
+    credentials = params.get('credentials')
+    client_id = credentials.get('identifier')
+    client_secret = credentials.get('password')
     proxy = params.get('proxy', False)
     verify_certificate = not demisto.params().get('insecure', False)
+    first_fetch = datetime.timestamp(dateparser.parse(params.get('first_fetch'))) if params.get('first_fetch') else None
     base_url = params.get('base_url')
     tlp_color = params.get('tlp_color')
-    include_deleted = argToBoolean(params.get('include_deleted', False))
+    include_deleted = params.get('include_deleted', False)
```

Comment:
Are you sure we don't need casting to bool here?

Reply:
No, it worked without casting, I checked it :)